### PR TITLE
Attempt to run acceptance tests without jquery using ember canary

### PR DESCRIPTION
**Pull request by cibernox**
After https://github.com/emberjs/ember.js/pull/15065 was merged in master, lets see if acceptance tests run without jquery.

### CODE REVIEW

**Comment by pixelhandler** [comment]
@cibernox looks like there is an error about a jQuery reference, see failed travis tests, `TypeError: _emberViews.jQuery is not a function`

**Comment by cibernox** [comment]
Those erros have to be fixed upstream in ember. I'll try to find some.

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
   },
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.5",
-    "ember-ajax": "^2.4.1",
+    "ember-ajax": "^2.5.6",
```
Comment (pixelhandler) [inline package.json]: could we remove this as a dev dependency?

Comment (cibernox) [inline package.json]: I was experimenting. I believe that ember-ajax is the problem here, but at the same time mirage seems to depend on jquery implicitly. 
This problem is a several layers' cake, but I'll get to the bottom of it.

**Comment by cibernox** [comment]
@rwjblue @pixelhandler It's alive! 

We can write apps and run its tests without using jQuery at all. 
Steps:

- Use canary (perhaps beta works too?)
- Install `ember-native-dom-event-dispatcher`
- Obviously use `ember-fetch` instead of `ember-ajax`.


This uses `async/await` all the way down, both integration and acceptance.

**Pull request merged**